localenv/cloud-nine-wallet/docker-compose.yml

```diff
@@ -110,7 +122,7 @@ services:
     image: rafiki-frontend
     build:
       context: ../..
-      dockerfile: ./packages/frontend/devDockerfile
+      dockerfile: ./packages/frontend/Dockerfile.dev
```

Comment:
went ahead and renamed this for consistency. I went with Dockerfile.dev because that seemed to be a more standard convention from examples I saw a variety of places. VSCode also recognizes its a dockerfile from name alone, whereas it did not with `devDockerfile`.